setup.py

```diff
@@ -2,27 +2,39 @@
 
 import codecs
 import os
-from distutils.core import setup
+from setuptools import setup
+import pathlib
 
-version = '?'
+version = '1.7.4+1.0.0'
```

Comment:
I think adding a version here is not good since I don't want to manually update it here. This is currently injectsed like this:
https://github.com/OpenRoberta/robertalab-ev3dev/blob/develop/debian/rules#L12

maybe it would be better to use None as a default and fail if it not importable from the environment.